async-http-client-backend/fs2/src/main/scala/com/softwaremill/sttp/asynchttpclient/fs2/AsyncHttpClientFs2Backend.scala

```diff
@@ -79,6 +79,7 @@ object AsyncHttpClientFs2Backend {
     AsyncHttpClientFs2Backend[F](client, closeClient = false)
 }
 
+// TODO replace with EffectMonadAsyncError when cats-effect versions are bin compat
 private[fs2] class EffectMonad[F[_]](implicit F: Effect[F]) extends MonadAsyncError[F] {
```

Comment:
are there conflicting dependencies between fs2 and the cats backend?

Reply:
Yeah, `cats-effect` 0.8.0 vs 1.0.0-RC1

```
sbt:sttp> asyncHttpClientBackendFs2/dependencyTree
[info] com.softwaremill.sttp:async-http-client-backend-fs2_2.12:1.1.15-SNAPSHOT [S]
[info]   +-com.github.zainab-ali:fs2-reactive-streams_2.12:0.5.1 [S]
[info]   | +-co.fs2:fs2-core_2.12:0.10.1 [S]
[info]   | | +-org.typelevel:cats-effect_2.12:0.8 [S]
...
```

```
sbt:sttp> cats/dependencyTree
[info] com.softwaremill.sttp:cats_2.12:1.1.15-SNAPSHOT [S]
[info]   +-com.softwaremill.sttp:core_2.12:1.1.15-SNAPSHOT [S]
[info]   +-org.typelevel:cats-effect_2.12:1.0.0-RC [S]
...
```

`fs2` 0.10.4 depends on `cats-effect` 0.10.0, but I don't think it would be safe to assume 0.10.0 is binary compatible with 1.0.0-RC1

Reply:
Ah, I thought fs2 updated already. Thanks :)